metadata/src/main/java/org/apache/kafka/controller/BrokersToElrs.java

```diff
@@ -159,4 +160,12 @@ BrokersToIsrs.PartitionsOnReplicaIterator partitionsWithBrokerInElr(int brokerId
         }
         return new BrokersToIsrs.PartitionsOnReplicaIterator(topicMap, false);
     }
+
+    BrokersToIsrs.PartitionsOnReplicaIterator partitionsWithElr() {
```

Comment:
This isn't acceptable from a performance point of view, since it will be O(num_partitions)